sagemaker-featurestore/feature_store_kms_key_encryption.ipynb

```diff
@@ -0,0 +1,554 @@
+{
```

Comment:
<p>you are creating a key on a SageMaker infrastructure, this means your sagemaker service role has root access to the key (if you allow it create key in the first place). I am not sure if this is a good practice, because intuitively you can think of a service role as your agent, not sure if you should let your agent to have full access to your key. But "best practice" is determined based on the specific use case, so I would suggest you say something about the how much power the sagemaker service role has on this key if the cell below went through. </p><p> </p><p>Second thing, "arn:aws:cloudtrail:*:123456789012:trail/*" is not a real account, you need to make a comment on that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I added more context to clarify this.